### ISSUE TITLE
DA-3 training information

### ISSUE THREAD

**Issue by nam1410** [ByteDance-Seed/Depth-Anything-3#251]
Dear authors,
Thanks for your great work! Do you have any plans to release the training code or more detailed implementation information for Depth-Anything-3? I’m interested in reproducing your results. Thank you,

**Comment by anhquancao** [comment]
Hi! Not one of the authors, but in case it helps anyone here. We implemented the full training pipeline and losses for DA3 following the paper details. Hopefully is is a useful starting point:

https://github.com/valeoai/OccAny#occany-depth-anything-3--sam3-1

More context in https://github.com/ByteDance-Seed/Depth-Anything-3/issues/35#issuecomment-4825419485
